schema/parser.go

```diff
@@ -373,3 +374,53 @@ func detectCharset(table sqlparser.TableSpec) string {
 	// TODO: consider returning err when charset is missing
 	return ""
 }
+
+type IdentityOpt struct {
+	Behavior string
+	Sequence string
+}
+
+func parseIdentityOpt(opt *sqlparser.IdentityOpt) *IdentityOpt {
+	if opt == nil {
+		return nil
+	}
+	dest := &IdentityOpt{
+		Behavior: strings.ToUpper(opt.Behavior),
+	}
+	if opt.Sequence != nil {
```

Comment:
I think building the `Sequence` string from various options is responsibility of generator.go. Could you change the type of `schema.IdentityOpt.Sequence` from `string` to a `schema.Sequence` struct that looks similar to `sqlparser.Sequence` (but with simpler field types), and move this logic to generator.go?

Reply:
changed ! 3ace2b11a2aa551c8f2519f7f2ac27b15244afdc